intg-denonavr/avr.py

```diff
@@ -853,9 +853,14 @@ def _increase_expected_volume(self):
 
     def _decrease_expected_volume(self):
         """Without telnet, decrease expected volume and send update event."""
-        if not self._use_telnet or self._expected_volume is None:
+        if self._telnet_healthy or self._expected_volume is None:
             return
         self._expected_volume = max(self._expected_volume - self._volume_step, 0)
         # Send updated volume if no update task in progress
         if not self._update_lock.locked():
             self._event_loop.create_task(self._receiver.async_update())
+
+    @property
+    def _telnet_healthy(self) -> bool:
```

Comment:
Note: this helper function could also be used in `async_update_receiver_data()`

Reply:
Do you mean to replace `_telnet_was_healthy` with the new property completely?